lldb/test/API/lang/swift/async/expr/TestSwiftAsyncExpressions.py

```diff
@@ -16,6 +16,20 @@ class TestSwiftAsyncExpressions(lldbtest.TestBase):
     def test_actor(self):
         """Test async unwind"""
         self.build()
+        self.do_test_actor()
+
+    @swiftTest
+    @skipIfWindows
+    @skipIfLinux
+    @skipIf(archs=no_match(["arm64", "arm64e", "arm64_32", "x86_64"]))
+    def test_actor_backdeploy(self):
+        """Test async unwind"""
+        self.build(dictionary={
+            'TARGET_SWIFTFLAGS': f'-target {self.getArchitecture()}-apple-macos11'
```

Comment:
I'm curious if this is better than using `SWIFTFLAGS_EXTRAS`. Are the makefiles themselves defining SWIFTFLAGS_EXTRAS?

Reply:
Yes the Makefiles are defining `SWIFTFLAGS_EXTRAS`. I believe it didn't work when I tried. I can double check.